<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>987</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> * {
font-family: 'Helvetica', open-sans;
}
    QLabel, QPushButton, QComboBox, QLineEdit {
    font-size: 25px;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Image">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>351</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>321</width>
      <height>918</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Name">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Name">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_M">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Helvetica,open-sans</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font-size: 35px;
}</string>
         </property>
         <property name="text">
          <string>М</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_F">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Helvetica,open-sans</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font-size: 35px;
}</string>
         </property>
         <property name="text">
          <string>Ж</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_Race">
       <property name="text">
        <string>Раса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Race">
       <item>
        <property name="text">
         <string>ЭЛЬФ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ФЕЯ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ОРК</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ЧЕЛОВЕК</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Strength">
       <property name="text">
        <string>Сила</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Strength"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Agility">
       <property name="text">
        <string>Ловкость</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Agility"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Intelligence">
       <property name="text">
        <string>Интеллект</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Intelligence"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Luck">
       <property name="text">
        <string>Удача</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Luck"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Points">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 33px;
}</string>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Create">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Clear">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SaveToTXT">
       <property name="text">
        <string>Сохранить (.txt)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SaveAsToTXT">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сохранить как (.txt)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_LoadFromTXT">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Загрузить (.txt)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SaveToBIN">
       <property name="text">
        <string>Сохранить (.bin)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SaveAsToBIN">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сохранить как (.bin)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_LoadFromBIN">
       <property name="font">
        <font>
         <family>Helvetica,open-sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Загрузить (.bin)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>580</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_Class">
         <property name="text">
          <string>Класс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_ClassValue">
         <item>
          <property name="text">
           <string>ЦЕЛИТЕЛЬ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ГУБИТЕЛЬ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ЗАЩИТНИК</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>БАНДИТ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ВОРЮГА</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>БАРБИ</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_Health">
         <property name="text">
          <string>Здоровье</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_HealthValue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_Mana">
         <property name="text">
          <string>Мана</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ManaValue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_Defence">
         <property name="text">
          <string>Защита</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_DefenceValue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_Attack">
         <property name="text">
          <string>Атака</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_AttackValue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
